rest/handler.go

```diff
@@ -283,6 +278,33 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 		}
 	}
 
+	// Named collections handling
+	if dbc := h.server.GetDbConfig(keyspaceDb); dbc != nil && len(dbc.Scopes) > 0 {
+		// Allow an empty scope to refer to the one SG is running with, rather than falling back to _default
+		if keyspaceScope == nil {
+			keyspaceScope = dbContext.BucketSpec.Scope
```

Comment:
Similar question here - why BucketSpec.Scope as opposed to checking in dbContext.Scopes?  If the intention is to design with support for multiple Scopes on the dbContext, then should add a way to specify the 'default' scope (not to be confused with _default scope) for the database.